### DISCUSSION TITLE
Time out for pac cli inputs: solution import in Azure DevOps

### DISCUSSION THREAD

**Discussion by eamedlab** [microsoft/powerplatform-build-tools#805] [General]
I have several DevOps pipelines setup using pac solution import that on occasion, appear to time out. I thought this was due to solution size, but reviewing today, I'm seeing that the import process is successful, the log on Azure lags behind, and stops communicating causing the step failure. When this happens, I edit the pipeline/release to skip the step and move forward, however, I'd like the process to finish automatically.

Why would the communication to DevOps be lagged to the point that an import would be successful 20 minutes earlier, but the DevOps step times out?

**Comment by ollitheorca** [comment]
We encounter the same issue almost every night when we build our solution. We extract from our development environment and import into a dedicated deployment environment to ensure that each new release works with the current version. From the "Deployment" pipeline, we then create our solution file as managed, which proceeds to testing and eventually to official release.

However, the import to the "Deployment" environment times out, causing the pipeline stages to break and preventing the creation of our solution file. As a result, we need to rerun this specific pipeline until it no longer times out. It is very frustrating, to say the least.

**Comment by jbuenoliveira** [comment]
@eamedlab  / @ollitheorca  - The `Overwrite unmanaged customizations` might be the issue, uncheck it and try again.

**Remarks**
This setting applies to scenarios where the managed solution is already installed and it is being installed again, perhaps as a newer version. When this value is `false` importing the solution will only update the managed solution components and any unmanaged customizations of those components will remain. By setting this property to `true` and importing the solution, any unmanaged customizations will be overwritten. You may want to do this as a way to re-initialize the solution.
